app/lib/theme/widgets/ConditionalAmountWidget.tsx

```diff
@@ -0,0 +1,90 @@
+import { WidgetProps } from "@rjsf/core";
+import { useEffect, useState } from "react";
+import RadioButton from "@button-inc/bcgov-theme/RadioButton";
+import NumberFormat from "react-number-format";
+import Grid from "@button-inc/bcgov-theme/Grid";
+
+export const ConditionalAmountWidget: React.FC<WidgetProps> = (props) => {
+  const { schema, id, label, onChange, value } = props;
+  const [requireAmount, setRequireAmount] = useState(false);
+  const [amount, setAmount] = useState(value);
+
+  const handleRequireAmount = () => {
+    setRequireAmount(true);
+    setAmount(0);
+    onChange(0);
+  };
+
+  const handleAmountNotRequired = () => {
+    setRequireAmount(false);
+    setAmount("");
+    onChange("");
+  };
+
+  useEffect(() => {
+    setAmount(value);
+  }, [value]);
+
+  return (
+    <div>
+      <Grid cols={10}>
+        <Grid.Row className="radio-button-row" align={"center"}>
+          <Grid.Col span={4}>
```

Comment:
all you need is a flexbox for your first option, right? When using a grid, you're creating (at least) 5 extra unnecessary `div`s